video-analysis/_posts/2017-04-03-PCA.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: review
+title:  "PCA: A Bayesian Computer Vision System for Modeling Human Interactions"
+tags:   motion-detection segmentation PCA
+author: Yi Wang
+pdf:    https://pdfs.semanticscholar.org/21ce/a8f56a0d067d640f923b2d69e18ed5542f6d.pdf
+cite:
+  authors: "Oliver, Nuria M and Rosario, Barbara and Pentland, Alex P"
+  title:   "A Bayesian Computer Vision System for Modeling Human Interactions"
+  venue:   "PAMI 2000"
+---
+
+Principal Component Analysis (PCA) is a matrix decomposition method for motion detection. For motion detection, PCA is based on the assumption that the moving objects are typically small and do not appear in the same location in a video. Because of that, the foreground objects do not have a significant contribution to background, and they will not be included in the most important eigenvectors when we apply matrix decomposition on the video. 
+
+By considering the whole video as a 3D matrix, PCA first apply singular value decomposition on it to find the eigenvalues and the eigenvectors of it. The eigenvectors are ranked according to their correlated eigenvalues. The top M eigenvectors are selected to be the eigenbackground space. When a new frame arrives, it will be first mean normalized, and then projected to the eigenbackground space to calculate the background image. After that, the foreground is calculated as the thresholding result of the difference between the input image and the background image.
```

Comment:
`the eigenvectors of it.` -> `the eigenvectors.` repetition, just before you say `on it`